Automattic/wp-calypso
# ET/FSE: Remove the "Showcase New Blocks" module

fixes #46530

#### Changes proposed in this Pull Request

* Remove the "Showcase New Blocks" module
Added in #41822
Requested for removal in #46530 ()

#### Testing instructions

* Have sandbox set up and refreshed
* `cd apps/editing-toolkit`
* `yarn run dev --sync`
* Visit sandbox and edit post
* Look in left side inserter for "New" section - should be gone

![2020-10-21_16-41](https://user-images.githubusercontent.com/937354/96790508-87ea7100-13bc-11eb-8043-6f805e0aba17.png)
![2020-10-21_16-41_1](https://user-images.githubusercontent.com/937354/96790518-8ae56180-13bc-11eb-8e6e-0cd9503143af.png)
